src/serverlist.cpp

```diff
@@ -297,7 +311,7 @@ void CServerListManager::OnTimerPollList()
     // server entry) and the predefined servers if they are still valid.
     // Note that we have to use "ServerList.size()" function in the for loop
     // since we may remove elements from the server list inside the for loop.
-    for ( int iIdx = 1 + iNumPredefinedServers; iIdx < ServerList.size(); iIdx++ )
+    for ( int iIdx = 1 + iNumPredefinedServers; iIdx < ServerList.size(); )
```

Comment:
Extra fix...

Reply:
Can you please give some more info on what was wrong here?

Reply:
`removeAt(i);` then `i++` -- it means you skip an entry in the list.

Reply:
Ups... Good that you found that.